dist/index.html

```diff
@@ -1,42 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>To Do List</title>
-  <script defer src="main.js"></script></head>
-  <body>
-    <main>
-      <header>
-        <div class="header">
-          <div class="todo">
-            <h1>Today's To Do</h1>
-            <div class="image">
-              <img src="./assets/reload.svg" alt="Reload Icon" class="reload" />
```

Comment:
- [ ] **Required:** Some weird bugs 🐛🐞 makes the images in this file not load. Please fix them. 😄

Reply:
Swap on awesome.